services/QuillLMS/app/models/activity.rb

```diff
@@ -128,6 +128,14 @@ class Activity < ApplicationRecord
   OLD_ELL_STARTER_DIAGNOSTIC_ACTIVITY_ID = 1161
   OLD_ELL_INTERMEDIATE_DIAGNOSTIC_ACTIVITY_ID = 1568
   OLD_ELL_ADVANCED_DIAGNOSTIC_ACTIVITY_ID = 1590
+
+  STARTER_PRE_DIAGNOSTIC_IDS = [STARTER_DIAGNOSTIC_ACTIVITY_ID, OLD_STARTER_DIAGNOSTIC_ACTIVITY_ID]
```

Comment:
Not in scope for this, but given that it sounds like we may be switching diagnostics again this summer, we'll probably want to change the name of the `OLD` ones to reflect their specific time scope.